### PR TITLE
Wip v2.01

### PR DESCRIPTION
**Pull request by maxwellbenton**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,16 +1,208 @@
...
+# Same as above, but using whitespace to make it easier for humans to read
+
+spice_rack_pretty = [
+  ["Mace", "Ginger", "Marojam"],
+  ["Paprika", "Fajita Mix", "Coriander"],
+  ["Parsley", "Sage", "Rosemary"]
+]
+```
+
+Since Ruby doesn't care about whitespace, so we'd advise you to write a 2-D
```
Comment (maxwellbenton) [inline README.md]: remove 'so' or 'Since'

File: lib/nested_array.rb [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+]
+
+def assembled_matrix
+  # Build an array that contains (or, "nests") the residents of The Ennet House
+  # and the Enfield Tennis Academy as provided by the constants
+end
+
+def array_literal_matrix
+  # Using Array literal syntax only, build a nested array that uses the data in
+  # side the ENNET_HOUSE and ENFIELD_TENNIS_ACADEMY Arrays but sorts
```
Comment (maxwellbenton) [inline lib/nested_array.rb]: inside

File: spec/nested_array_spec.rb [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+describe 'When a learning to update cells in nested arrays,' do
+  describe 'when given a matrix, row, column, and new_value argument,' do
+    it 'the matrix_update method returns an updated matrix' do
+      local_copy_of_matrix = assembled_matrix.dup
+      result = matrix_update(local_copy_of_matrix, 0, 2, "Tiny Ewell")
+      expect(result[0]).to eq(["Don Gately", "Joelle van Dyne", "Tiny Ewell", "Kate Gompert", "Bruce Green"])
+    end
+  end
+end
```
Comment (maxwellbenton) [inline spec/nested_array_spec.rb]: solution checked, looks good

File: README.md [hunk 4]
```diff
@@ -1,16 +1,208 @@
...
+![Chessboard with Stamma's Notation](https://curriculum-content.s3.amazonaws.com/programming-univbasics-5/nested-arrays-lab/chess_grid.png)
+
+Think about latitude and longitude uniquely identifying a place on a map.
+
+![Map + LatLong](https://curriculum-content.s3.amazonaws.com/programming-univbasics-5/nested-arrays-lab/nyc_latlong_sm.png)
+
+_Photo credit: Mike C. Valdivia on Unsplash_
+
+
+### From Array to Array of Array
```
Comment (drakeltheryuujin) [inline README.md]: This headline might be a little confusing. Certain plays on words might not be very friendly to a global audience (it took me a couple rereads to understand).

File: README.md [hunk 5]
```diff
@@ -1,16 +1,208 @@
...
+
+An `Array` is like a list but in code form. It is a way for your program to
```
Comment (drakeltheryuujin) [inline README.md]: "list, but in code..." (add comma)

File: README.md [hunk 6]
```diff
@@ -1,16 +1,208 @@
...
+store pieces of data as elements in a collection. Arrays can contain any
+combination of data types -- `Boolean`s, `Integer`s, `Strings`.
+
+Let's imagine an `Array` as a **single** long shelf with books on it. The shelf
+is like the `Array` and each book is an element of it.  We can refer to books
+by saying things like the following:
+
```
Comment (drakeltheryuujin) [inline README.md]: I think a graphics request from Rob here would be super useful

**Pull request merged**